contracts/contracts/rem.system/include/rem.system/rem.system.hpp

```diff
@@ -540,10 +553,12 @@ namespace eosiosystem {
          global_state2_singleton _global2;
          global_state3_singleton _global3;
          global_state4_singleton _global4;
+         global_state5_singleton _global5;
          eosio_global_state      _gstate;
          eosio_global_state2     _gstate2;
          eosio_global_state3     _gstate3;
          eosio_global_state4     _gstate4;
+         eosio_global_state5     _gstate5;
          rammarket               _rammarket;
```

Comment:
please start a separate PR to clean up legacy variables like this. (+ legacy actions and commented code)

Reply:
AFAIK _rammarket is not a legacy variable